migrations/00008_targeted_messaging_add_outreach_1/index.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO outreaches (
+    source_id,
+    name,
+    start_date,
+    end_date,
+    type,
+    team_name,
+    source_file,
+    source_file_checksum
+) VALUES (
+    1,
+    '',
+    '1970-01-01 00:00:00.000',
+    '1970-01-01 00:00:00.000',
+    '',
+    '',
```

Comment:
These are just placeholders for non-nullable columns. The actual data will be loaded as soon as the JSON file is processed (at the service startup).